ros/lunar/debian/stretch/ros-core/Dockerfile

```diff
@@ -14,24 +14,17 @@ RUN apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys 4B63CF8FDE
 # setup sources.list
 RUN echo "deb http://snapshots.ros.org/lunar/final/debian stretch main" > /etc/apt/sources.list.d/ros1-snapshots.list
 
-# install bootstrap tools
-RUN apt-get update && apt-get install --no-install-recommends -y \
-    python-rosdep \
-    python-rosinstall \
-    python-vcstools \
-    && rm -rf /var/lib/apt/lists/*
-
 # setup environment
 ENV LANG C.UTF-8
 ENV LC_ALL C.UTF-8
 
-# bootstrap rosdep
-RUN rosdep init \
-    && rosdep update --include-eol-distros
+ENV ROS_DISTRO lunar
+
+# label ros packages
+LABEL sha256.ros-lunar-ros-core=cdf4fa8a555d13e8d69b7bf2fd69d8169d36949d40ee7700a642b9a125f77204
```

Comment:
Something @mikaelarguedas mentioned to me. Is this the same sha the is in the snapshot repo?

Reply:
If we decide to not update EOL images (suggested in https://github.com/osrf/docker_images/pull/454#issuecomment-689149674) we should end up with all the Dockerfiles using snapshots.ros.org remaining unchanged in this PR.

If we decide to update them anyways, we should check that the sha is matching the one from snapshots.ros.org.